dashboard/test/ui/support/blockly_helpers.rb

```diff
@@ -11,14 +11,14 @@ def generate_drag_code(from, to, target_dx, target_dy)
   end
 
   def generate_selector_drag_code(from, to, target_dx, target_dy)
-    "var drag_dx = $(\"#{to}\").position().left - $(\"#{from}\").position().left;" +
-        "var drag_dy = $(\"#{to}\").position().top  - $(\"#{from}\").position().top;" +
+    "var drag_dx = $(\"#{to}\").offset().left - $(\"#{from}\").offset().left;" +
+        "var drag_dy = $(\"#{to}\").offset().top  - $(\"#{from}\").offset().top;" +
         "$(\"#{from}\").simulate( 'drag', {handle: 'corner', dx: drag_dx + #{target_dx}, dy: drag_dy + #{target_dy}, moves: 5});"
   end
 
   def generate_begin_to_drag_code(from, to, target_dx, target_dy)
-    "var drag_dx = $(\"[block-id='#{to}']\").position().left - $(\"[block-id='#{from}']\").position().left;" +
-        "var drag_dy = $(\"[block-id='#{to}']\").position().top  - $(\"[block-id='#{from}']\").position().top;" +
+    "var drag_dx = $(\"[block-id='#{to}']\").offset().left - $(\"[block-id='#{from}']\").offset().left;" +
+        "var drag_dy = $(\"[block-id='#{to}']\").offset().top  - $(\"[block-id='#{from}']\").offset().top;" +
         "$(\"[block-id='#{from}']\").simulate( 'drag', {justDrag: true, handle: 'corner', dx: drag_dx + #{target_dx}, dy: drag_dy + #{target_dy}, moves: 5});"
   end
```

Comment:
This change has the possibility of breaking other UI tests, though I think we ought to be okay. I'll make sure things pass before committing.